### PR TITLE
Add drush `cim` and `cex`.

### PR DESCRIPTION
**Pull request by mediaashley**
@joemewes, this is completely untested, so please don't make any assumptions.

### CODE REVIEW

File: src/Command/Drush/DrushConfigExportCommand.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+use Docker\Drupal\Extension\ApplicationContainerExtension;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Docker\Drupal\Style\DruDockStyle;
+
+/**
+ * Class DemoCommand
```
Comment (joemewes) [inline src/Command/Drush/DrushConfigExportCommand.php]: name class `DrushConfigExportCommand`

File: src/Command/Drush/DrushConfigExportCommand.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * @file
+ * Contains \Docker\Drupal\Command\DemoCommand.
```
Comment (joemewes) [inline src/Command/Drush/DrushConfigExportCommand.php]: name class DrushConfigExportCommand

**Comment by joemewes** [comment]
@mediaashley gonna merge (think should be into develop but I haven't "mastered" the git flow plan for this repo.). Will update comments locally and test.  have a couple other recently merged PRs to go into release with this. BIG THANKS! #communtiycontributions :)

**Comment by mediaashley** [comment]
Yeah, I though it should probably be develop, but I could only see PRs in to master. I figgured you'd upadate as you thought best ;)

I'm always missing those comments. My PHPStorm setup automatically colapses them, and then I forget :( I usually catch it when I commit though…

**Comment by joemewes** [comment]
Also, when you create new commands you need to "register" them in the Application like
https://github.com/4AllDigital/DruDockCli/blob/master/src/Application.php#L187

all merged and released. depending on what version you're on, you should be able to run `drudock self:update` and it'll output v 1.4-alpha1.0.8.

**Comment by joemewes** [comment]
also... read PHP comments ... you're not the only one.....https://github.com/4AllDigital/DruDockCli/commit/7e3c1e07a1068078c915ead2d5ef03f7365acd7f

**Comment by mediaashley** [comment]
What's the "right" way for me to test locally/move to the latest tag? (I originally installed using the phar as per the ReadMe)

**Pull request merged**